apps/montreal_elixir/lib/montreal_elixir/social_feeds/meetup_api_client.ex

```diff
@@ -0,0 +1,68 @@
+if Mix.env == :test || Mix.env == :travis do
```

Comment:
Let's discuss.  I wonder if introducing a "Fake" MeetupApiClient would be a good alternative to requiring an Elixir script, like I do in my Twitter integration.  Not sure though.

Reply:
I looked at your PR briefly and "Fake" would be a good way for integration tests. However, for MeetupApiClient unit tests I want to make sure the :httpc.request/1 is actually called. Yeah, lets talk tonight :)